### DISCUSSION TITLE
spatial extension?

### DISCUSSION THREAD

**Discussion by nshiab** [duckdb/duckdb-wasm#1531] [General]
Hi!

Do you have an example of how to use the [spatial extension](https://duckdb.org/docs/archive/0.9.2/extensions/spatial) with duckdb-wasm?

Thank you very much. :)

**Comment by carlopi** [comment]
```sql
LOAD spatial;

CREATE TABLE nyc AS
    SELECT
        borough,
        st_union_agg(geom) AS full_geom,
        st_area(full_geom) AS area,
        st_centroid(full_geom) AS centroid,
        count(*) AS count FROM
st_read('https://raw.githubusercontent.com/duckdb/duckdb_spatial/main/test/data/nyc_taxi/taxi_zones/taxi_zones.shp')
GROUP BY borough;

SELECT borough, area, centroid::VARCHAR, count FROM nyc;
```
on the live https://shell.duckdb.org is expected to work.

**Reply by kbatchu** [reply]
I understand that the spatial extension is not included in duckdb-wasm. Are there any plans to include the spatial extension in duckdb-wasm anytime soon?
Thanks,
